src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/stats/FlowStatSpec.groovy

```diff
@@ -97,7 +97,17 @@ class FlowStatSpec extends HealthCheckSpecification {
             assert pathHelper.convert(newFlowPathInfo) == currentProtectedPath
             assert pathHelper.convert(newFlowPathInfo.protectedPath) == currentPath
         }
-        def timeAfterSwap = new Date().getTime()
+
+        and: "Wait till stats from old main path are collected"
+        Wrappers.wait(WAIT_OFFSET, 3) {
+            statsHelper."force kilda to collect stats"()
+            def newStats = flowStats.of(flow.getFlowId())
+            newStats.get(FLOW_RAW_BYTES, srcSwitchId, mainForwardCookie).getNewestTimeStamp() ==
```

Comment:
this needs an explicit `assert` keyword. Otherwise it's not really checking anything, is it?

Reply:
Good catch.